ansible/roles/test/tasks/bgp_speaker.yml

```diff
@@ -77,13 +77,26 @@
     - {file_name: "config_3.ini", local_ip: '{{vlan_ips[0]}}', port_num: '7000'}
   delegate_to: "{{ptf_host}}"
 
+- set_fact: addr_family='ipv4'
+  when: addr_family is not defined
+
+- name: Activate IPv6 address family
+  shell: vtysh -c "conf t" -c "router bgp {{ minigraph_bgp_asn }}" -c "address-family ipv6" -c "neighbor BGPSLBPassive activate"
+  when: addr_family == 'ipv6'
```

Comment:
please move this into the bgpd.conf.j2 in buildimage.

Reply:
It is already in buildimage (hash commit 127a73a), so removed these changes.